data_structures/src/data_request.rs

```diff
@@ -83,8 +86,14 @@ impl DataRequestPool {
                         dr_state.info.reveals.values().cloned().collect();
                     reveals.sort_unstable_by_key(|reveal| {
                         if active_wips.wip0019() {
-                            // TODO: Hash(pkh+dr)
-                            PublicKeyHash::default()
+                            let mut bytes_to_hash = vec![];
+                            bytes_to_hash.extend(reveal.body.pkh.hash);
+                            bytes_to_hash.extend(dr_pointer.as_ref());
+
+                            let hashed_bytes = calculate_sha256(bytes_to_hash.as_ref()).0;
+
+                            // We are forcing 20 bytes, so we can not obtain an error here
+                            PublicKeyHash::from_bytes(&hashed_bytes[..20]).unwrap()
```

Comment:
You can avoid this hack by moving the `if active_wips.wip0019()` outside, so that `sort_unstable_by_key` is inside the if.